Orm/Xtensive.Orm/Orm/Linq/Translator.Expressions.cs

```diff
@@ -33,19 +33,29 @@ namespace Xtensive.Orm.Linq
 {
   internal sealed partial class Translator
   {
+    private static readonly ParameterExpression parameterContextParam = Expression.Parameter(WellKnownOrmTypes.ParameterContext, "context");
+    private static readonly ConstantExpression
+      nullKeyExpression = Expression.Constant(null, WellKnownOrmTypes.Key),
+      falseBoolExpression = Expression.Constant(false, WellKnownTypes.Bool),
+      trueBoolExpression = Expression.Constant(true, WellKnownTypes.Bool),
+      falseExpression = Expression.Constant(false),
+      trueExpression = Expression.Constant(true);
```

Comment:
Why expression for the same constants are declared twice?

Reply:
Original code had usage of both forms.
Now I see they are semantically equal.

Removed duplication